api/tools/src/main/java/org/pmiops/workbench/tools/LoadDataDictionary.java

```diff
@@ -30,7 +32,7 @@
   private Resource[] getDataDictionaryExportFiles() throws IOException {
```

Comment:
I'd name this `getDataDictionaryResources` so you don't give callers the knowledge that it's implemented with flat files.